src/dfx/src/commands/canister/install.rs

```diff
@@ -103,12 +105,15 @@ pub async fn exec(
                 )?;
                 let wasm_module = dfx_core::fs::read(wasm_path)?;
                 let mode = mode_hint.to_install_mode_with_wasm_path()?;
-                info!(
-                    env.get_logger(),
-                    "{} code for canister {}",
-                    install_mode_to_prompt(&mode),
-                    canister_id,
+                let spinner = env.new_spinner(
```

Comment:
This doesn't play well with the "ask for consent" and interactive parameter prompting in that can happen when installing a canister. To repro, for example:

```
$ dfx canister install --wasm /Users/ericswanson/d/sdk/e2e/assets/wasm/identity/main.wasm --mode reinstall react_rust_backend
⠁ Reinstalling code for canister react_rust_backend, with canister ID bkyz2-fmaaa-aaaaa-qaaaq-cai                           WARNING!
You are about to reinstall the react_rust_backend canister
This will OVERWRITE all the data and code in the canister.

YOU WILL LOSE ALL DATA IN THE CANISTER.


Do you want to proceed? yes/No
⠋ Reinstalling code for canister react_rust_backend, with canister ID bkyz2-fmaaa-aaaaa-qaaaq-cai  
```